ext/opentelemetry-ext-jaeger/README.rst

```diff
@@ -56,7 +56,7 @@ gRPC is still not supported by this implementation.
     span_processor = BatchExportSpanProcessor(jaeger_exporter)
 
     # add to the tracer
-    tracer.add_span_processor(span_processor)
+    trace.tracer_source().add_span_processor(span_processor)
```

Comment:
There was a change between 0.4 and the current master to rename `TracerSource` to `TraceProvider`. For this example to work with the master version, this code will need to be updated to show:
```suggestion
    trace.tracer_provider().add_span_processor(span_processor)
```

Or alternatively, to work with 0.4. the code above will need to be updated to use `TracerSource`
https://github.com/open-telemetry/opentelemetry-python/pull/459/files#diff-2f282c97a47c5c1e4db9466939a1721bR38

Reply:
ops, I already had forgotten about that one.

Reply:
Good to know. Given the rapid development, I suggest just updating master.